server/mergin/sync/public_api_v2_controller.py

```diff
@@ -43,3 +44,32 @@ def delete_project_now(id_):
     db.session.commit()
 
     return NoContent, 204
+
+
+@auth_required
+def update_project(id_):
+    """Rename project
+
+    :param id_: Project_id
+    :type id_: str
+    """
+    project = require_project_by_uuid(id_, ProjectPermissions.Update)
+    new_name = request.json["name"]
+
+    if not is_name_allowed(new_name):
+        return (
+            jsonify(
+                code="InvalidProjectName", detail="Entered project name is invalid"
```

Comment:
I think we should stick with existing form of returning customized errors (this is where we aimed migrate to)
https://github.com/MerginMaps/mergin/blob/develop/server/mergin/sync/errors.py

Reply:
ok, let's deal with this in separate PR with errors clean up